scripts/prune.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+
+// Description
+
+// Prune  validatorMessages from the database
+// An optional timestam can be passed or it prunes HeartBeat messages
+// that are less than the current date and uses adexValidator as its default database
+//
+//
+// Database
+// ----------
+// Default database it connects to is `adexValidator`
+// but can be overwritten via the `DB_MONGO_NAME` environment variable
+//
+//
+// Options
+// ------------------------------------
+// channelId (required)
+// timestamp ( default = current date ) e.g. 2015-01-01
+// all (required) (default = adexValidator)
+//
+//
+//
+// Example
+// ----------------------------------------
+//
+// Prune validator messages from an expired channel in database X
+// DB_MONGO_NAME='x' ./scripts/prune.js --channelId='expiredChannel'
+//
+// Prune validator messages from a specific date
+// ./sccripts/prune.js --timestamp='2012-01-01' --channelId='testing'
+//
+// Delete validator Messages for epxired channel
+// ./sccripts/prune.js testValStackLeader1558782672 --channelId='testing'
+//
+// Prune validator messages for all expired channels
+// ./scripts/prune.js --all
+//
+//
+//
+
+const assert = require('assert')
+const yargs = require('yargs')
+const db = require('../db')
+const logger = require('../services/logger')('prunning')
+
+const { argv } = yargs
+	.usage('Usage $0 [options]')
+	.option('channelId')
+	.describe('channelId', 'channelId to prune')
+	.option('timestamp')
+	.describe('timestamp', 'timestamp to prune heartbeat messages')
+	.default('timestamp', new Date().toISOString())
+	.boolean('all')
+	.describe('all', 'delete validator messages for all expired channels')
+
+async function start() {
+	db.connect()
+		.then(async () => {
+			if (argv.all) {
+				await pruneAll()
+			} else {
+				await run()
+			}
+			process.exit()
+		})
+		.catch(err => {
+			logger.error(err.message)
+		})
+}
+
+async function run() {
```

Comment:
add an arguments `channelId` ad `timestamp` here, and rename to `pruneChannel`

then, we should have a new function `pruneAll` that will call `pruneExpired` (current `pruneAll`) and `pruneChannel` for each non-expired channel